libs/order/testing/src/factories/order-item.factory.ts

```diff
@@ -23,16 +23,17 @@ export class MockOrderItem implements DaffOrderItem {
   sku = 'sku';
   name = 'Product Name';
   weight = faker.random.number({min: 1, max: 1000});
-  qty = faker.random.number({min:1, max:100});
+  qty = faker.random.number({min: 1, max: 10});
   price = faker.random.number({min: 1, max: 1000});
-  discount_percent = faker.random.number({min: 1, max: 10});
-  discount_amount = faker.random.number({min: 1, max: 100});
+  discount_amount = faker.random.number({min: 1, max: this.price});
+  discount_percent = Math.floor(this.discount_amount / this.price * 100);
```

Comment:
This should be `Math.floor(this.discount_amount / this.price) * 100`

Reply:
Wouldn't that always be 0?